src/index.jsx

```diff
@@ -243,18 +248,23 @@ export default class FormulaBar extends Component {
     );
     let valueLast;
     let insideFunc = false;
-
-    if (valueTail.match(/[A-Za-z]+\d*$/g)) {
-      [valueLast] = valueTail.match(/[A-Za-z\d]+$/g).reverse();
-    } else if (valueTail.match(/[A-Za-z]+\d*\([^\)]*/g)) {
+    const valueLastMatch1 = valueTail.match(/[A-Za-z\d]+$/g);

```

Comment:
It seems like this code should be the same as `suggestionWorker` but it's a little bit different.

It would be better to be able to extract it out.